backend/config/default.yaml

```diff
@@ -1,6 +1,10 @@
+# production configuraiton
 application:
   name: project-group-careless-crayfish
   mongo:
     address: 127.0.0.1:27017
-    username : devroot
-    password : devroot
\ No newline at end of file
+    username: devroot
```

Comment:
I can't get the npm config library working with `docker-compose` either in CI env or running it locally.

So I have instead actually been loading these configs/credentials from a `.env` file under the `backend` folder...